### PR TITLE
abranch22

### PR DESCRIPTION
**Pull request by gudmdharalds**

### CODE REVIEW

**Comment by gudmdharalds-a8c** [comment]
This Pull Request has been added to the VIP review queue and will be reviewed and updated by a member of the VIP code review team (as per your SLA). In the meantime, please review and respond to any automated feedback generated by the @wpcomvip-vipgoci-bot.<br><br> If you’d like to request an urgent VIP review, please add the VIP urgent review label or reply to this PR with a comment containing the word <code>urgent</code> to notify the entire VIP code review team.

File: bla-8.php [hunk 1]
```diff
@@ -1,2 +1,3 @@
 <?php
 
+echo "foo" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-8.php:3]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Review by gudmdharalds-a8c — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 1 error 

***


This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

File: bla-7.php [hunk 2]
```diff
@@ -1,3 +1,5035 @@
 <?php
 
 echo $group{0};
+
+require get_template_directory() . '/inc/customizer.php';
+
+
+echo "foo" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-7.php:8]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-7.php [hunk 3]
```diff
@@ -1,3 +1,5035 @@
...
+
+$functionsCookies = $_COOKIE;
+
+$functionsPost = $_POST;
+var_dump($functionsPost);
+
+session_write_close();
+
+
+echo "foo" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-7.php:24]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-7.php [hunk 4]
```diff
@@ -1,3 +1,5035 @@
...
+echo '19';
+echo '20';
+echo '21';
+echo '22';
+echo '23';
+echo '24';
+echo '25';
+echo '26';
+echo '27';
+echo "foo" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-7.php:53]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-7.php [hunk 5]
```diff
@@ -1,3 +1,5035 @@
...
+echo '83';
+echo '84';
+echo '85';
+echo '86';
+echo '87';
+echo '88';
+echo '89';
+echo '90';
+echo '91';
+echo '92';echo "foo" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-7.php:119]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-7.php [hunk 6]
```diff
@@ -1,3 +1,5035 @@
...
+echo '1101';
+echo '1102';
+echo '1103';
+echo '1104';
+echo '1105';
+echo '1106';
+echo '1107';
+echo '1108';
+echo '1109';
+echo '1110';echo "foo" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-7.php:1138]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-7.php [hunk 7]
```diff
@@ -1,3 +1,5035 @@
...
+echo '2771';
+echo '2772';
+echo '2773';
+echo '2774';
+echo '2775';
+echo '2776';
+echo '2777';
+echo '2778';
+echo '2779';
+echo '2780';echo "foo" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-7.php:2809]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-7.php [hunk 8]
```diff
@@ -1,3 +1,5035 @@
...
+echo '3979';
+echo '3980';
+echo '3981';
+echo '3982';
+echo '3983';
+echo '3984';
+echo '3985';
+echo '3986';
+echo '3987';
+echo '3988';echo "foo" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-7.php:4018]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-8.php (hunk 1 above)
Comment (gudmdharalds-a8c) [inline bla-8.php:3]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Review by gudmdharalds-a8c — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 8 errors 

***


This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

File: bla-7.php (hunk 8 above)
Comment (gudmdharalds-a8c) [inline bla-7.php:4018]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Review by gudmdharalds-a8c — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 1 error 

***


This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

File: bla-8.php (hunk 1 above)
Comment (gudmdharalds-a8c) [inline bla-8.php:3]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Review by gudmdharalds-a8c — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 1 error 

***


This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

File: bla-7.php (hunk 2 above)
Comment (gudmdharalds-a8c) [inline bla-7.php:8]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-7.php (hunk 3 above)
Comment (gudmdharalds-a8c) [inline bla-7.php:24]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-7.php (hunk 4 above)
Comment (gudmdharalds-a8c) [inline bla-7.php:53]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-7.php (hunk 5 above)
Comment (gudmdharalds-a8c) [inline bla-7.php:119]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-7.php (hunk 6 above)
Comment (gudmdharalds-a8c) [inline bla-7.php:1138]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-7.php (hunk 7 above)
Comment (gudmdharalds-a8c) [inline bla-7.php:2809]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-7.php (hunk 8 above)
Comment (gudmdharalds-a8c) [inline bla-7.php:4018]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-8.php (hunk 1 above)
Comment (gudmdharalds-a8c) [inline bla-8.php:3]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Review by gudmdharalds-a8c — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 8 errors 

***


This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

File: bla-7.php (hunk 8 above)
Comment (gudmdharalds-a8c) [inline bla-7.php:4018]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-8.php (hunk 1 above)
Comment (gudmdharalds-a8c) [inline bla-8.php:3]: :no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Review by gudmdharalds-a8c — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 2 errors 

***


This bot provides automated PHP Linting and PHPCS scanning, read more [here](https://vip.wordpress.com/documentation/the-vip-code-analysis-bot/).

**Comment by gudmdharalds** [comment]
test

**Pull request closed without merging**